pipeline/filters/kubernetes.md

```diff
@@ -204,3 +206,149 @@ Under certain and not common conditions, a user would want to alter that hard-co
 
 So at this point the filter is able to gather the values of _pod\_name_ and _namespace_, with that information it will check in the local cache \(internal hash table\) if some metadata for that key pair exists, if so, it will enrich the record with the metadata value, otherwise it will connect to the Kubernetes Master/API Server and retrieve that information.
 
+## Optional Feature: Using Kubelet to Get Metadata
```

Comment:
the explanation here is good. May be ignore my other comment on the config option and instead have a link in the description to this section.

Reply:
Added the link inside this section.